### PR TITLE
Add connectivity site references

### LINKED ISSUE #218: Link to libp2p connectivity site
## Done Criteria

- [ ] We have a link to the new `connectivity.libp2p.io` site

---

## Open Question

Should it be added as a it's own section (on the left nav bar), under Reference, or linked elsewhere?
@DannyS03 @marten-seemann your preference?

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Adds references to the new connectivity site: connectivity.libp2p.io
- Added a simple reference section doc.
- Added to left nav bar.
- Added ending sentence on the transport index.
- closes #218 

## Latest preview

Please view the latest Fleek preview [here](https://bafybeieqpsuyt7tz7un5pyiqsljtevziytx35m7wweuygrjwsojgrezgv4.on.fleek.co/).

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Hugo build was successful!
found div [<div id="body-inner">
<h1 id="connectivity-in-libp2p">Connectivity in libp2p</h1>
<p>Check out the <a href="https://connectivity.libp2p.io/">connectivity site</a> for types of connectivity in libp2p.</p>
<footer class="footline">
</footer>
</div>]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Hugo build was successful!
found div [<div id="body-inner">
<h1 id="exchange-bytes">Exchange Bytes</h1>
<p>When you make a connection from your computer to a machine on the internet,
chances are pretty good you’re sending your bits and bytes using TCP/IP, the
wildly successful combination of the Internet Protocol, which handles addressing
and delivery of data packets, and the Transmission Control Protocol, which
ensures that the data that gets sent over the wire is received completely and in
the right order.</p>
<p>Because TCP/IP is so ubiquitous and well-supported, it’s often the default
choice for networked applications. In some cases, TCP adds too much overhead,
so applications might use <a href="https://en.wikipedia.org/wiki/User_Datagram_Protocol">UDP</a>,
a much simpler protocol with no guarantees about reliability or ordering.</p>
<p>While TCP and UDP (together with IP) are the most common protocols in use today,
they are by no means the only options. Alternatives exist at lower levels
(e.g. sending raw ethernet packets or bluetooth frames), and higher levels
(e.g. QUIC, which is layered over UDP).</p>
<p>The foundational protocols that move bits around are called transports, and one of
libp2p’s core requirements is to be transport agnostic. This means that the decision
of what transport protocol to use is up to the developer, and an application can support
many different transports at the same time. Learn about the fundamentals of transport
protocols in libp2p.</p>
<p>To better visualize the types of libp2p connections, check out the
<a href="https://connectivity.libp2p.io/">connectivity site</a> which outlines the different types
of connectivity in libp2p.</p>
<ul class="children children-li">
<li>
<a href="../../concepts/transports/listen-and-dial/">Listening and Dialing</a>
</li>
<p>Transports are defined in terms of two core operations, listening and dialing. Learn about how peers use both interfaces in libp2p.</p>
<li>
<a href="../../concepts/transports/quic/">QUIC</a>
</li>
<p>QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed connection built on top of UDP. Learn about QUIC and how it is used in libp2p.</p>
<li>
<a href="../../concepts/transports/webtransport/">WebTransport</a>
</li>
<p>WebTransport is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p.</p>
<li>
<a href="../../concepts/transports/webrtc/">WebRTC</a>
</li>
<p>WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p.</p>
</ul>
<footer class="footline">
</footer>
</div>]
found div [<div id="body-inner">
<h1 id="connectivity-in-libp2p">Connectivity in libp2p</h1>
<p>Check out the <a href="https://connectivity.libp2p.io/">connectivity site</a> for types of connectivity in libp2p.</p>
<footer class="footline">
</footer>
</div>]

File: config.toml [hunk 1]
```diff
@@ -33,5 +33,10 @@ name = "<i class='fab fa-discourse'></i> Discussion Forums"
 url = "https://discuss.libp2p.io"
 weight = 4
 
+[[menu.shortcuts]]
+name = "<i class='fab fa-discourse'></i> Connectivity"
```
Comment (p-shahi) [inline config.toml:37]:
```suggestion
name = "<i class='fab fa-book'></i> Connectivity"
```

It's not a discourse page, I suggest we use book.
A better choice is browser: https://fontawesome.com/icons/browser?s=solid&f=classic but I don't know what fontawesome version this site uses

**Review by p-shahi — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Hugo build was successful!
found div [<div id="body-inner">
<h1 id="exchange-bytes">Exchange Bytes</h1>
<p>When you make a connection from your computer to a machine on the internet,
chances are pretty good you’re sending your bits and bytes using TCP/IP, the
wildly successful combination of the Internet Protocol, which handles addressing
and delivery of data packets, and the Transmission Control Protocol, which
ensures that the data that gets sent over the wire is received completely and in
the right order.</p>
<p>Because TCP/IP is so ubiquitous and well-supported, it’s often the default
choice for networked applications. In some cases, TCP adds too much overhead,
so applications might use <a href="https://en.wikipedia.org/wiki/User_Datagram_Protocol">UDP</a>,
a much simpler protocol with no guarantees about reliability or ordering.</p>
<p>While TCP and UDP (together with IP) are the most common protocols in use today,
they are by no means the only options. Alternatives exist at lower levels
(e.g. sending raw ethernet packets or bluetooth frames), and higher levels
(e.g. QUIC, which is layered over UDP).</p>
<p>The foundational protocols that move bits around are called transports, and one of
libp2p’s core requirements is to be transport agnostic. This means that the decision
of what transport protocol to use is up to the developer, and an application can support
many different transports at the same time. Learn about the fundamentals of transport
protocols in libp2p.</p>
<p>To better visualize the types of libp2p connections, check out the
<a href="https://connectivity.libp2p.io/">connectivity site</a> which outlines the different types
of connectivity in libp2p.</p>
<ul class="children children-li">
<li>
<a href="../../concepts/transports/listen-and-dial/">Listening and Dialing</a>
</li>
<p>Transports are defined in terms of two core operations, listening and dialing. Learn about how peers use both interfaces in libp2p.</p>
<li>
<a href="../../concepts/transports/quic/">QUIC</a>
</li>
<p>QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed connection built on top of UDP. Learn about QUIC and how it is used in libp2p.</p>
<li>
<a href="../../concepts/transports/webtransport/">WebTransport</a>
</li>
<p>WebTransport is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p.</p>
<li>
<a href="../../concepts/transports/webrtc/">WebRTC</a>
</li>
<p>WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p.</p>
</ul>
<footer class="footline">
</footer>
</div>]
found div [<div id="body-inner">
<h1 id="connectivity-in-libp2p">Connectivity in libp2p</h1>
<p>Check out the <a href="https://connectivity.libp2p.io/">connectivity site</a> for types of connectivity in libp2p.</p>
<footer class="footline">
</footer>
</div>]

**Pull request merged**